testsuite/features/support/commonlib.rb

```diff
@@ -37,7 +37,7 @@ def compute_channels_to_leave_running
     os_family = node.os_family
     next unless ['sles', 'rocky'].include?(os_family)
     os_version = os_version.split('.')[0] if os_family == 'rocky'
-    raise "Can't build list of reposyncs to leave running" unless %w[12-SP4 12-SP5 15-SP3 15-SP4 8].include? os_version
+    log "Can't build list of reposyncs to leave running" unless %w[15-SP3 15-SP4 8].include? os_version
```

Comment:
That will probably fail.
log is a method provided and handled by Cucumber, so it can only be used inside the step definitions, but not in plain ruby functions.
If it works... well, I learnt something new :)

Reply:
We already use this in other plain Ruby files. See e.g. https://github.com/uyuni-project/uyuni/blob/master/testsuite/features/step_definitions/api_common.rb#LL490C4-L490C4

Reply:
Good point, I'll run a test to check if that somehow works, otherwise I'll revert to using `raise`

Reply:
I wonder if it is really loading the classpath of the Cucumber log https://github.com/cucumber/cucumber-ruby/pull/1523, or it's another kind of log implementation provided by Ruby 🤔

Reply:
@nodeg the file you shared and the line you shared, is inside a step definition.

Reply:
Ok, now I see what you meant Oscar.

Reply:
Just did a test run while inducing the error, the `log` does seem to be working:
![image](https://user-images.githubusercontent.com/100688791/222476225-57e28c20-79ed-40df-8eb5-c0eab53396fe.png)